### ISSUE TITLE
Optimize automation playbooks by executing tasks in parallel

### ISSUE THREAD

**Issue by spisarski** [cablelabs/transparent-security#53]
There are a number of playbooks that are performing some tasks that can be done in parallel with others speeding up our automation scripts.

This is more of an EPIC so please create a new ticket when attempting to optimize a section of any of our playbooks and add the ticket and PR numbers/links on this issue.

**Comment by spisarski** [comment]
Automation has been tweaked about as much as we should do for now.
